libs/designer/src/lib/ui/settings/index.tsx

```diff
@@ -234,15 +234,31 @@ function GeneralSettings({ nodeId, readOnly }: { nodeId: string; readOnly?: bool
     updateOutputsAndTokens(nodeId, operationInfo, dispatch, isTrigger, nodeInputs, { ...settings, splitOn: splitOnSetting });
   };
 
+  const onInvokerConnectionToggle = (checked: boolean): void => {
+    dispatch(
+      updateNodeSettings({
+        id: nodeId,
+        settings: {
+          invokerConnection: {
+            isSupported: !!invokerConnection?.isSupported,
+            value: { value: invokerConnection?.value?.value ?? undefined, enabled: checked },
```

Comment:
```suggestion
            value: { value: invokerConnection?.value?.value, enabled: checked },
```

This means the same thing